tools/issue-labeler/src/IssueLabelerService/Answers/OpenAiAnswerService.cs

```diff
@@ -175,22 +175,23 @@ private async Task RetrieveAndAggregateSubqueryResults(
             int top,
             double scoreThreshold,
             Dictionary<string, string> labels,
-            HashSet<string> uniqueIssues,
-            List<Issue> aggregatedIssues,
-            HashSet<string> uniqueDocs,
-            List<Document> aggregatedDocs)
+            HashSet<Issue> uniqueIssues,
+            HashSet<Document> uniqueDocs
+        )
         {
             var subqueryTasks = subqueries.Select(async subquery =>
             {
                 var subqueryIssues = await _ragService.SearchIssuesAsync(issueIndexName, issueSemanticName, issueFieldName, subquery, top, scoreThreshold, labels);
                 var subqueryDocs = await _ragService.SearchDocumentsAsync(documentIndexName, documentSemanticName, documentFieldName, subquery, top, scoreThreshold, labels);
-                AddUniqueItems(subqueryIssues, uniqueIssues, aggregatedIssues, issue => issue.Id);
-                AddUniqueItems(subqueryDocs, uniqueDocs, aggregatedDocs, doc => doc.Url);
+                foreach (var subqueryIssue in subqueryIssues)
+                    uniqueIssues.Add(subqueryIssue);
+                foreach (var subqueryDoc in subqueryDocs)
+                    uniqueDocs.Add(subqueryDoc);
```

Comment:
```suggestion
                foreach (var subqueryIssue in subqueryIssues)
                {
                    uniqueIssues.Add(subqueryIssue);
                }
                
                foreach (var subqueryDoc in subqueryDocs)
                {
                    uniqueDocs.Add(subqueryDoc);
                }
```

nit: Always use braces, even for single lines.  It helps add safety and eliminate a class of errors.